bdsf/islands.py

```diff
@@ -182,7 +182,7 @@ def ndimage_alg(self, img, opts):
         img.island_labels = labels
 
         ### apply cuts on island size and peak value
-        pyrank = N.zeros(image.shape, dtype=N.int32)
+        pyrank = N.zeros(image.shape, dtype=int)
```

Comment:
Here you double the size of `pyrank`.

Reply:
Here the array is larger, so it might be more important (though I don't think it will make any practical difference here either). I will change all the occurrences back to their original values just to be on the safe side.